File: nodejs/single-number-136.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let xor = 0;

    for (let x of nums) {
        xor ^=  x;
    }

    return xor;
};

console.log(singleNumber([2,3,4,6, 2,3,4]));

File: golang/singlenumber136/single-number-136.go
package main

import "fmt"

func main() {
	fmt.Println("Hello")
	inp := [...]int{2, 3, 5, 9, 2, 4, 3, 5, 9}

	singleNumber(inp[0:len(inp)])
}

func singleNumber(nums []int) int {
	xor := 0
	for _, v := range nums {
		//fmt.Println(v)
		xor = xor ^ v
	}
	fmt.Println(xor)
	return xor
}
